frontend/src/app/main/ui/routes.cljs

```diff
@@ -94,10 +95,17 @@
 (defn on-navigate
   [router path]
   (let [location (.-location js/document)
+        [base-path qs] (str/split path "?")
+
+        qstring
```

Comment:
maybe we can use the app.common.uri/query-string->map helper that already does this